modules/wordpress-com-block-editor-iframe.php

```diff
@@ -42,3 +42,76 @@ function jetpack_add_iframed_body_class( $classes ) {
 	return $classes;
 }
 add_filter( 'admin_body_class', 'jetpack_add_iframed_body_class' );
+
+/**
+ * Enqueue the scripts for the WordPress.com block editor integration.
+ */
+function jetpack_enqueue_wpcom_block_editor_scripts() {
+	$suffix        = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+	$version       = gmdate( 'YW' );
+	$is_calypsoify = 1 === (int) get_user_meta( get_current_user_id(), 'calypsoify', true );
+
+	wp_enqueue_script(
+		'wpcom-block-editor-common',
+		'//widgets.wp.com/wpcom-block-editor/common' . $suffix . '.js',
+		array( 'lodash', 'wp-compose', 'wp-data', 'wp-editor', 'wp-rich-text' ),
+		$version,
+		true
+	);
+
+	wp_localize_script(
+		'wpcom-block-editor-common',
+		'wpcomGutenberg',
+		array(
+			'switchToClassic' => array(
+				'isVisible' => false,
+			),
+			'isCalypsoify'    => $is_calypsoify,
+			'richTextToolbar' => array(
+				'justify'   => __( 'Justify' ),
+				'underline' => __( 'Underline' ),
+			),
+		)
+	);
+
+	if ( $is_calypsoify ) {
+		wp_enqueue_script(
+			'wpcom-block-editor-calypso-iframe-bridge',
+			'//widgets.wp.com/wpcom-block-editor/calypso-iframe-bridge-server' . $suffix . '.js',
+			array( 'calypsoify_wpadminmods_js', 'jquery', 'lodash', 'react', 'wp-blocks', 'wp-data', 'wp-hooks', 'wp-tinymce', 'wp-url' ),
+			$version,
+			true
+		);
+	}
+}
+add_action( 'admin_enqueue_scripts', 'jetpack_enqueue_wpcom_block_editor_scripts' );
```

Comment:
Maybe we could only enqueue it on the pages where we need it? I think that would be nice given the number of dependencies we are calling here.

Reply:
@mmtr To save you some leg work, `admin_enqueue_scripts` passes `$hook_suffix` that we can test for `post.php` and `post-new.php`. I believe those are the two pages for editing an existing post and creating a new one.

Alternatively we could also use `enqueue_block_editor_assets`, if that is not too late in the execution path.

Reply:
I'd agree with @jeherve . I'm not sure `get_current_screen()` is reliable yet at this point, but I think a `$pagenow` check for either `post.php` or `post-new.php` would work anyways.

Reply:
Using `enqueue_block_editor_assets` did the trick. Thanks @obenland for the suggestion!